README.md

```diff
@@ -4,6 +4,8 @@ Kubernetes Charts Repository for CS3ORG
 
 This reposity contains the official charts for CS3Org software, like Reva.
 
+See also [configuring an iop deployment](https://developer.sciencemesh.io/docs/technical-documentation/iop/deployment/kubernetes/#configuring-an-iop-deployment).
```

Comment:
Can you move this line towards the example at the end? I think it makes semantically much more sense to include there, next to the revad installation command.

Reply:
Yes! Done.